src/Capability/Discovery/DocBlockParser.php

```diff
@@ -136,4 +136,51 @@ public function getParamTypeString(?Param $paramTag): ?string
 
         return null;
     }
+
+    /**
+     * Gets the return type string from a Return tag.
+     */
+    public function getReturnTypeString(?DocBlock $docBlock): ?string
+    {
+        if (!$docBlock) {
```

Comment:
always like to be explicit
```suggestion
        if (null === $docBlock) {
```